store/service_store.go

```diff
@@ -59,6 +59,9 @@ type ServiceManager interface {
 	// Noted that the return block number may be different from the input one because
 	// we don't have state in the input one.
 	GetERC20Balance(ctx context.Context, contractAddress, address common.Address, blockNr int64) (*decimal.Decimal, *big.Int, error)
+
+	// Subscriptions store
+	FindTotalBalance(blockNumber int64, token common.Address, group int64) (result *model.TotalBalance, err error)
```

Comment:
do M&M want to query all tokens together or one by one?
if former, you should pass a list of tokens and do everything in one sql stmt with `IN` clause.

Reply:
I think subscription store can support query many tokens even if M&M would like to query one by one.

Reply:
According to the previous discussion with M&M, they want to query one by one as the first priority for their existed logic (please refer to ES-132.) The second priority is to query all tokens once API then.